<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceWidget</class>
 <widget class="QWidget" name="DeviceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="1" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>设备参数</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3" rowstretch="1,0">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>系统参数</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="3" column="2" colspan="3">
           <widget class="QLineEdit" name="lineEdit_4"/>
          </item>
          <item row="8" column="2" colspan="2">
           <widget class="QDateTimeEdit" name="dateTimeEdit"/>
          </item>
          <item row="1" column="2" colspan="3">
           <widget class="QLineEdit" name="lineEdit_2"/>
          </item>
          <item row="5" column="2" colspan="3">
           <widget class="QLineEdit" name="lineEdit_6"/>
          </item>
          <item row="4" column="2" colspan="3">
           <widget class="QLineEdit" name="lineEdit_5"/>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>设备型号</string>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>系统时间</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2" colspan="3">
           <widget class="QLineEdit" name="lineEdit"/>
          </item>
          <item row="2" column="2" colspan="3">
           <widget class="QLineEdit" name="lineEdit_3"/>
          </item>
          <item row="9" column="2">
           <widget class="QPushButton" name="pushButton_2">
            <property name="text">
             <string>设置时间</string>
            </property>
           </widget>
          </item>
          <item row="9" column="3">
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>  恢复出厂设置</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>硬件版本</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>供电方式</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0" colspan="2">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>电量</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="5" column="0" colspan="2">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>设备ID</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>软件版本</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>数据配置</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="1">
           <widget class="QGroupBox" name="groupBox_5">
            <property name="title">
             <string>Y轴报警值</string>
            </property>
            <layout class="QFormLayout" name="formLayout_2">
             <item row="0" column="0">
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>加速度</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QSpinBox" name="spinBox_4"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>速度</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QSpinBox" name="spinBox_5"/>
             </item>
             <item row="2" column="1">
              <widget class="QSpinBox" name="spinBox_6"/>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_14">
               <property name="text">
                <string>振幅</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QGroupBox" name="groupBox_6">
            <property name="title">
             <string> Z轴报警值</string>
            </property>
            <layout class="QFormLayout" name="formLayout_3">
             <item row="0" column="0">
              <widget class="QLabel" name="label_15">
               <property name="text">
                <string>加速度</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QSpinBox" name="spinBox_7"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_16">
               <property name="text">
                <string>速度</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QSpinBox" name="spinBox_8"/>
             </item>
             <item row="2" column="1">
              <widget class="QSpinBox" name="spinBox_9"/>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_17">
               <property name="text">
                <string>振幅</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="pushButton_6">
            <property name="text">
             <string>数据纠偏</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QGroupBox" name="groupBox_7">
            <property name="title">
             <string>其它</string>
            </property>
            <layout class="QFormLayout" name="formLayout_4">
             <item row="0" column="0">
              <widget class="QLabel" name="label_18">
               <property name="text">
                <string>采样率</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QSpinBox" name="spinBox_10"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_19">
               <property name="text">
                <string>采样间隔</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QSpinBox" name="spinBox_11"/>
             </item>
             <item row="2" column="1">
              <widget class="QSpinBox" name="spinBox_12"/>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_20">
               <property name="text">
                <string>温度预警</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QGroupBox" name="groupBox_4">
            <property name="title">
             <string>X轴报警值</string>
            </property>
            <layout class="QFormLayout" name="formLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>加速度</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QSpinBox" name="spinBox"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>速度</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QSpinBox" name="spinBox_2"/>
             </item>
             <item row="2" column="1">
              <widget class="QSpinBox" name="spinBox_3"/>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>振幅</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QPushButton" name="pushButton_5">
            <property name="text">
             <string>设置参数</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>固件更新</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>固件路径</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_7"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_3">
            <property name="text">
             <string>打开文件</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_4">
            <property name="text">
             <string>开始更新</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>设备数据</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_21">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
